src/Console/Command/Notification.php

```diff
@@ -130,16 +137,19 @@ private function generateMessages($notification)
                     'message' => trans('notifications.' . $messageType . '.message', $subs),
                     'type' => $messageType,
                     'data_source' => $data_source,
```

Comment:
Need to make this $data_source->getId() otherwise we're trying to save an object and failing